src/web/topic/components/TopicWorkspace/MoreActionsDrawer.tsx

```diff
@@ -479,6 +498,46 @@ export const MoreActionsDrawer = ({
           </ToggleButton>
         </ListItem>
       </List>
+
+      {/* Screenshot Resolution Dialog  */}
+      <Dialog open={screenshotDialogOpen} onClose={() => setScreenshotDialogOpen(false)}>
+        <DialogTitle>Set Screenshot Resolution</DialogTitle>
+        <DialogContent>
+          <TextField
+            fullWidth
+            label="Width"
+            type="number"
+            margin="dense"
+            InputProps={{ endAdornment: <InputAdornment position="end">px</InputAdornment> }}
+            error={!!errors.width}
+            helperText={errors.width?.message}
+            {...register("width", { valueAsNumber: true, min: 1 })}
```

Comment:
```suggestion
            {...register("width", { valueAsNumber: true })}
```
Our zod schema's `positive()` handles this `min: 1` validation so it's not necessary here